tracks/elixir/exercises/roman-numerals/README.md

```diff
@@ -0,0 +1,79 @@
+### Reasonable solutions
+
+```elixir
+defmodule Roman do
+  @dict [
+    {1000, "M"},
+    {900, "CM"},
+    {500, "D"},
+    {400, "CD"},
+    {100, "C"},
+    {90, "XC"},
+    {50, "L"},
+    {40, "XL"},
+    {10, "X"},
+    {9, "IX"},
+    {5, "V"},
+    {4, "IV"},
+    {1, "I"}
+  ]
+
+  def numerals(0, acc), do: acc
```

Comment:
Looks great, you are missing the starting scenario `def numerals(number), do: do_numerals(number, "")`

Reply:
Nice catch! I had a default argument in the definition below, but removed it.